common/v2/features/AddAccount/components/Mnemonic.tsx

```diff
@@ -16,11 +18,11 @@ import questionToolTip from 'common/assets/images/icn-question.svg';
 import './Mnemonic.scss';
 
 interface OwnProps {
+  formData: FormData;
```

Comment:
Prefer passing the desired property. i.e `network`

Reply:
Same as above.

Reply:
closing because out of scope